cmd/function-sidecar_test.go

```diff
@@ -0,0 +1,106 @@
+package main_test
+
+import (
+	"testing"
+	"net/http"
+	"os/exec"
+	"os"
+	"fmt"
+	"time"
+	"gopkg.in/Shopify/sarama.v1"
+	"bufio"
+	"errors"
+)
+
+func TestIntegrationWithKafka(t *testing.T) {
+	fmt.Println("TestIntegrationWithKafka invoked")
+
+	SOURCE_MESSAGE_BODY := `{"key": "value"}`
+
+	broker := os.Getenv("KAFKA_BROKER")
+	if broker == "" {
+		t.Fatal("Required environment variable KAFKA_BROKER was not provided")
+	}
+
+	fmt.Println("Building function-sidecar")
```

Comment:
Would rather add integration tests to the Makefile and use dependencies there, wdyt?